src/Uno.UWP/ApplicationModel/DataTransfer/DataPackageOperation.cs

```diff
@@ -0,0 +1,14 @@
+namespace Windows.ApplicationModel.DataTransfer
+{
+
+  // only one operation is supported - copy; 'none' is default value in DataPackage
+	public   enum DataPackageOperation 
+	{
+		None,
+		Copy,
+		#if false
```

Comment:
This would cause a breaking change - the enum will have to be supported as a whole.

Reply:
I think that breaking build is better than allowing to build ok, but at runtime, works incorrectly. As Move or Link is unsupported (doesn't do anything)...
But, from the other hand, there is no clipboard.get operation, and DataPackageOperation maybe is relevant only for Get?

Reply:
@pkar70 That's not the usage philosophy of Uno. Instead the idea is that any code that builds for UWP/WinUI will also build with Uno, but the packaged analyzers will surface the usage of unsupported APIs by checking for the `[NotImplemented]` attribute.

Reply:
Enums cannot be partial, you have to copy the whole definition, and keep all the fields.

Reply:
Yes, that would work better in this case.